modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -0,0 +1,86 @@
+* The administrator of your organization's {prod-short} instance or the repository maintainer or the README author has provided you with the URL of your organization's {prod-short} instance that you need as a user to access your {prod-short} dashboard pages and start and manage your workspaces. This URL is a fully qualified domain name (FQDN): `{prod-url}`.
+* Your Git repository maintainer keeps the `devfile.yaml` or `.devfile.yaml` file in the root directory of the Git repository. (For alternative file names and file paths, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].)
++
+IMPORTANT: Although the devfile enables full functionality of a workspace, you can also start a new workspace by supplying the URL of a Git repository without a devfile. This results in a new workspace with a clone of the Git repository and with the defaults such as the Che-Theia IDE and the Universal Developer Image.
+//provide xrefs to a concept module about the Universal Developer Image and, if applicable, devfile-less defaults for new workspaces. max-cx
++
+TIP: Advanced users can compose and use their own devfiles.
+//add links on "compose" or "devfiles" to relevant sections when those are drafted. max-cx
+* Optional: Private repositories that are hosted on GitHub, GitLab, or Bitbucket require
+.. an authorized application link to be created on the repository-hosting website and
+.. a Kubernetes secret to be applied by the administrator of your organization's {prod-short} instance.
+//two inline links to be added into this par in the next, separate PR. max-cx
+//* "Kubernetes" rather than {platforms-name} requested by Mario. max-cx
+
+.Procedure
+
+. Optional: visit your {prod-short} dashboard page to authenticate as a {prod-short} user to your organization's instance of {prod-short}.
+
+. Visit the URL to start a new workspace using the basic syntax:
+[source,subs="+quotes,+attributes"]
++
+----
+{prod-url}#__<git_repository_url>__
+----
++
+[TIP]
+====
+You can extend this URL with optional parameters:
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?__<optional_parameters>__ <1>
+----
+<1> See xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].
+====
++
+.URL syntax for starting a new workspace that contains a clone of a GitHub-hosted repository
+====
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/tree/__<branch_name>__]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/pull/__<pull_request_id>__]` starts a new workspace with a clone of the branch of the pull request.
+====
++
+After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page in *Progress*.
```

Comment:
Did you mean...
```suggestion
After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page, *In Progress*.
```

Reply:
Generally, you don't need to describe widely-understood GUI behaviors unless they are a source of confusion and alarm. Even then, it's questionable whether documenting the problematic behavior is likely to help users. If you have to document something like this, that's usually a good sign that there's that you should log a usability bug for the interface...

Reply:
Yeah, I get what you mean. I'll drop *[In] Progress* from the sentence.